Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:6
+FROM debian:latest
```

Comment:
I still have a problem with this. If Debian 6 is no good I want to know why it's no good. I have no problem upgrading to a newer version if we need to but I want to make sure that we really do need to, so I first want to understand exactly why it doesn't work any more. 

If we decide an upgrade is necessary then the version should be pinned. I do not like the specifier being `latest` because it will change in future and may break the build. There is no reason why we should always have the latest distribution because we are not interested in the operating system we are just interested in having the necessary tools to run Buildroot.